### PR TITLE
feat(docs): add hacktoberfest post

### PR DESCRIPTION
**Pull request by pswu11**

### CODE REVIEW

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+<Screenshot
+  alt="QuestDB at Hacktoberfest 2022"
+  height={450}
+  src="/img/blog/2022-09-30/banner-hacktoberfest.png"
+  width={800}
+/>
+
+## Hacktoberfest
+
+For those who're not familiar with Hacktoberfest, it's a month-long online
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:38]: I don't have a good explanation for this but this sounds better to my ears:

```suggestion
For those who aren't familiar with Hacktoberfest, it's a month-long online
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+celebration for open-source software and communities. The first 40,000
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:39]: softwares or maybe "projects"?
```suggestion
celebration for open-source softwares and communities. The first 40,000
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+participants who
+[successfully completed the requirements](https://hacktoberfest.com/participation/#contributors)
+will be rewarded with a special-edition Hacktoberfest T-shirt 👕 or a tree
+planted in your name. 🌴
+
+Participating in Hacktoberfest is one of our approaches to raise awareness and
+encourage more developers or technical writers to contribute to open source. And
+to us, we welcome both code contributions and non-code contributions, such as
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:47]:
```suggestion
encourage more developers or technical writers to contribute to open source. We welcome both code contributions and non-code contributions, such as
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+   [our guidelinesm for docs contributors](https://github.com/questdb/questdb.io#contributing).
+3. [**Web console**](https://github.com/questdb/ui): Monorepo that contains web
+   console components.
+
+## 🚀 Contribute to QuestDB
+
+If you're interested in contributing to QuestDB, make sure you read the official
+guidelines about
+[contributors](https://hacktoberfest.com/participation/#contributors),
+[pull requests](https://hacktoberfest.com/participation/#pr-mr-details) and
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:82]:
```suggestion
[pull requests](https://hacktoberfest.com/participation/#pr-mr-details), and
```
You like Oxford comma right? ;p

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  the projects.
+- Avoid commenting on all the available issues but those you really plan to work
+  on, so you can leave some opportunities to other contributors.
+- Our maintainers will review your pull requests, please make sure you address
+  all the comments before asking for another review.
+
+## 🎁 QuestDB swag
+
+In addition to the official reward, if you successfully contribute one valid
+pull-request to QuestDB projects listed above, we offer an extra QuestDB T-shirt
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:102]:
```suggestion
pull-request to any of the QuestDB projects listed above, we offer an extra QuestDB T-shirt
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+for you through our [swag program](/community). 🚀
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:103]:
```suggestion
for you through our [swag program](/community/). 🚀
```
Is this correct?

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+<Screenshot
+  alt="QuestDB swag at Hacktoberfest 2022"
+  height={450}
+  src="/img/blog/2022-09-30/swag-hacktoberfest.png"
+  width={800}
+/>
+
+## ℹ️ Get help
+
+Some questions might appear when you're trying to contribute to QuestDB
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:114]:
```suggestion
If you have any questions when trying to contribute to QuestDB
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+- **StackOverflow**:
+  [https://stackoverflow.com/questions/tagged/questdb](https://stackoverflow.com/questions/tagged/questdb)
+
+Also, don't forget to follow us on social media to receive the latest updates:
+
+- **Twitter**: [https://twitter.com/questdb](https://twitter.com/questdb)
+- **Linkedin**:
+  [https://www.linkedin.com/company/questdb/](https://www.linkedin.com/company/questdb/)
+
+Lastly, star our [GitHub repo](https://github.com/questdb/questdb) if you
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:129]:
```suggestion
Last but not least, star our [GitHub repo](https://github.com/questdb/questdb) if you
```

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+  src="/img/blog/2022-09-30/swag-hacktoberfest.png"
+  width={800}
+/>
+
+## ℹ️ Get help
+
+Some questions might appear when you're trying to contribute to QuestDB
+projects; here are the places to get help from our team:
+
+- **Community Slack**: [https://slack.questdb.io](https://slack.questdb.io);
```
Comment (amyshwang) [inline blog/2022-09-30-hacktoberfest-questdb.md:117]: Is there a special reason why we don't do hyperlink here and below? (just curious!)

**Review by amyshwang — APPROVED** [review]
Mostly stylistic comments.

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+no dependencies and zero garbage collection. It is optimized for high-throughput
+ingestion over InfluxDB line protocol and fast SQL queries. QuestDB is also one
+of the most popular time series databases according to the independent reviewer
+[DBEngines](https://db-engines.com/en/ranking/time+series+dbms).
+
+Developers can use QuestDB as a library for java applications.
+[Official clients](/docs/reference/clients/overview/) for Python, Go, C, C++,
+Node.js, Rust, and .NET. are also available for the wider developer community.
+
+This year, there are three QuestDB open source projects opted in for
```
Comment (argshook) [inline blog/2022-09-30-hacktoberfest-questdb.md:63]: we have more than 3, maybe we include clients also?

https://github.com/orgs/questdb/repositories?q=client&type=public&language=&sort=

File: blog/2022-09-30-hacktoberfest-questdb.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+
+1. [**QuestDB**](https://github.com/questdb/questdb): QuestDB core database,
+   mainly written in Java and C++. Check
+   [CONTRIBUTING.md](https://github.com/questdb/questdb/blob/master/CONTRIBUTING.md)
+   to get started.
+2. [**Documentation**](https://github.com/questdb/questdb.io): QuestDB's website
+   for documentation, suitable for technical writers who're experienced in
+   Docusaurus and Markdown. Read
+   [our guidelinesm for docs contributors](https://github.com/questdb/questdb.io#contributing).
+3. [**Web console**](https://github.com/questdb/ui): Monorepo that contains web
```
Comment (argshook) [inline blog/2022-09-30-hacktoberfest-questdb.md:74]:
```suggestion
3. [**Web console**](https://github.com/questdb/ui/tree/main/packages/web-console): Monorepo that contains web
```

let's link straight to the `web-console` package

**Review by argshook — CHANGES_REQUESTED** [review]

File: blog/2022-09-30-hacktoberfest-questdb.md (hunk 10 above)
Comment (pswu11) [reply blog/2022-09-30-hacktoberfest-questdb.md:63]: for now they're not opted in yet and we don't have good ideas for issues. I'd leave it out for now - will collect some ideas from others for this next week.

**Review by argshook — APPROVED** [review]

**Pull request merged**